apps/studio/src/features/editing-experience/components/MoveResourceModal/MoveItem.tsx

```diff
@@ -1,33 +1,23 @@
-import { Suspense, useMemo } from "react"
+import { Suspense } from "react"
 import { Text } from "@chakra-ui/react"
 import { Button } from "@opengovsg/design-system-react"
 import { QueryErrorResetBoundary } from "@tanstack/react-query"
 import { ErrorBoundary } from "react-error-boundary"
-import { BiFile, BiFolder } from "react-icons/bi"
+import { BiFolder } from "react-icons/bi"
 
 import type { RouterOutput } from "~/utils/trpc"
 
 type MoveItemProps = Pick<
   RouterOutput["resource"]["getFolderChildrenOf"]["items"][number],
-  "permalink" | "type"
+  "permalink"
 > & {
   onChangeResourceId: () => void
 }
 
 const SuspendableMoveItem = ({
```

Comment:
suggestion: I duplicated the functionality of this into ResourceSelector and haven't got the time to merge the two, but I was thinking ideally they use the same one but in this MoveResourceModal, we disable selecting normal pages and only allow users to select folders.

Reason being I think it would be helpful for the user to know the pages that are also present in that particular folder. Might be a design decision so I can follow up with design if needed.

Reply:
i don't recall if it was a design decision tbh, but lets just ask @sehyunidaaa now HAHAHA

Reply:
<img width="625" alt="image" src="https://github.com/user-attachments/assets/838f5eb0-598f-4aca-96b1-abc0e14d6733">

This is a good suggestion! I have a state for folders that you don't have access to, maybe can use a similar state for pages so that we can still show them but disable selection?

Reply:
sounds good, lets agree on:
1. showing pages but they are disabled
2. not showing collections

just fyi, i fixed the caching but might need to recheck after adding pages in. are we ok with doing as a follow up PR?

Reply:
[chin/isom-1542-move-modal-should-also-show-lla](https://linear.app/ogp/issue/ISOM-1542/move-modal-should-also-show-pages)